template/src/MainLayout.js

```diff
@@ -14,21 +14,21 @@ function MainLayout() {
 
   const handleAppState = useCallback(() => {
     AppState.addEventListener('change', nextAppState => {
-      if (appStateRef.current.match(/inactive|background/) && nextAppState === 'active') {
-      }
+      // if (appStateRef.current.match(/inactive|background/) && nextAppState === 'active') {
+      // }
       appStateRef.current = nextAppState
     })
   }, [])
 
   const handleDeepLink = useCallback(() => {
     Linking.getInitialURL().then(res => {
       //This function only work when disable debug mode.
-      if (res) {
-      }
+      // if (res) {
+      // }
     })
     Linking.addEventListener('url', res => {
-      if (res?.url) {
-      }
+      // if (res?.url) {
+      // }
```

Comment:
Why we need comment this? if the eslint warning empty code block please add comment inside.

Reply:
Done